server/.env.example

```diff
@@ -144,3 +146,13 @@ PARTICLE_WEBHOOK_API_KEY_TEST=webhookkeytest
 # Unless specifically investigating something, this should be off (false)
 IS_DB_LOGGING=false
 IS_DB_LOGGING_TEST=false
+
+# The client ID of PA (for use in brave-alert-lib)
+# Get this from Google Cloud --> Brave PA Sign-In --> APIs & Services --> Credentials --> Web ID / Web ID (dev)
```

Comment:
Great. Thanks for repeating this comment about where to find the appropriate values here.